framework-cloud/framework-cloud-api/src/main/java/org/kie/cloud/api/settings/builder/KieServerS2ISettingsBuilder.java

```diff
@@ -52,6 +52,14 @@
     KieServerS2ISettingsBuilder withControllerUser(String controllerUser, String controllerPwd);
 
     /**
+     * Return configured builder with connection to Controller.
+     *
+     * @param protocol Protocol for URL to Controller.
+     * @return Builder
+     */
+    KieServerS2ISettingsBuilder withControllerProtocol(String protocol);
```

Comment:
What protocols do we support? http/https/ws? I'd prefer using Enum in this case..

Reply:
Updated